introduction.md

```diff
@@ -1,4 +1,4 @@
 Just trying and learning things.
-Javascript
-rebase
-lalalal
+* Javascript
+* rebase
+* lalalal
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?